### ISSUE TITLE
1.16: Timed messages do not repeat.

### ISSUE THREAD

**Issue by mibby** [kangarko/ChatControl#423]
Minecraft Version: 1.16.4
MySQL: true
BungeeCord: false

ChatControl Red 10.1.12

I'm running into an issue migrating from ChatControl 8 where broadcast messages aren't repeating. I'm not sure if it's just not repeating normally or if it's because ``/chc reload`` broke the repeat.

timed.rs
```
group global
prefix &6&l(!)&r
messages:
- &eExample message 1
- &eExample message 2
- &eExample message 3
- &eExample message 4
- &eExample message 5
- &eExample message 6
- &eExample message 7
- &eExample message 8
- &eExample message 9
- &eExample message 10
- &eExample message 11
- &eExample message 12
- &eExample message 13
```
settings.yml
```
# -------------------------------------------------------------------------------------------------
# Represents settings for player messages such as join, quit, death and timed messages
# -------------------------------------------------------------------------------------------------
Messages:
 
  # What aspects of messages should be enabled?
  # Options: join, quit, kick, death, timed
  #
  # If you have another plugin handling join/quit etc. messages, remove them from list below.
  # If you want to hide those messages, leave them in Apply_On and empty your messages/ .rs files.
  Apply_On:
  - 'join'
  - 'quit'
  - 'timed'
  
  # Automatically append prefix to messages of the given type. Edit/remove/add sections below
  # from Apply_On key above to the left, and prefix to the right.
  # The message types that you do not specify below, will have no prefix unless specified
  # individually. You can also disable prefix for some messages by writing "prefix none" there.
  #
  # Example:
  # Prefix:
  #   join: "&8[&2+&8] &7"
  #   quit: "&8[&4-&8] &7"
  #   kick: "&8[&4-&8] &7"
  #   death: "&8[&4x&8] &7"
  #   timed: "{announce_prefix}"
  Prefix:
  
  # Broadcast messages automatically to your Discord.
  # Put the types from Apply_On above to the left, and the Discord channel to the right. Your
  # Discord channel must first be connected in the Integration section below!
  #
  # Example:
  # Discord:
  #   join: standard
  Discord:
  
  # If you want to display join, quit and kick messages after other plugins, you can send them
  # a bit later such as 1 tick later than the other messages. Customize this here. 
  # **YOU CANNOT OVERRIDE THIS WITH THE "DELAY" OPERATOR**
  Player_Defer_Sending_By: 1 tick
  
  # The default broadcast delay. You can override this for each broadcast with the "delay" operator
  # as long as it is above this limit.
  Timed_Delay: 240 seconds
```

**Comment by ElBananaa** [comment]
Hey!
I'm not sure about that, however, I believe you need to add `delay 240 seconds` under the `group global`
Let me know if it fixed your issue.

**Comment by kangarko** [comment]
@ElBananaa nope that's not by this, in fact adding this is totally unneeded because if you don't have this, then the Timed_Delay default is used.

<img width="397" alt="Screenshot 2020-12-30 at 19 14 25" src="https://user-images.githubusercontent.com/1918426/103372790-4a4c4780-4ad3-11eb-9b9c-06d6ea45ba42.png">

Restart the server and use the next upcoming version, it will work.

**Comment by kangarko** [comment]
If still having issues by then, I need to see your /chc debug zip file

**Comment by mibby** [comment]
Confirmed seems to be fixed as of Red 10.2.1, thanks!
